pytest_redis/config.py

```diff
@@ -33,7 +33,7 @@ def get_conf_option(option: str) -> Any:
         return request.config.getoption(option_name) or request.config.getini(option_name)
 
     port = get_conf_option("port")
-    if (modules := get_conf_option("modules")) is not None:
+    if (modules := get_conf_option("modules")):
```

Comment:
the parenthesis left at the moment is what black complains about :)

Reply:
🤦🏻‍♂️ sorry

Reply:
Should be fine now